### ISSUE TITLE
DRAFT: Automatically add registered users who opted-in for newsletter to CRM

### ISSUE THREAD

**Issue by dboyer** [open-apparel-registry/open-apparel-registry#1320]
# Draft

This is a draft issue, and it will need to clarify specs and formalize what
counts as "done" before work begins on it.

Check with the issue's creator before starting work on this issue.

## Overview

Still deciding on CRM. Only users who have selected the "sign up for OAR newsletter" box at registration will be fed into the CRM.

### Additional Context

Optional. Add any other context, screenshots, gifs, etc about the feature request here.

**Comment by dboyer** [comment]
Closing in favor of more detailed issue in /clients
